### ISSUE TITLE
training on my own model

### ISSUE THREAD

**Issue by aluds123** [wizyoung/YOLOv3_TensorFlow#198]
Can you guide me how to train my own model(I modify the layers, like add maxpool layer...)?
I train it but mAP only get "0.09", and it also seems to be overfitting.The loss become small(0.4) on training, but the eval loss become bigger(39).

I also train on YOLOv3 model by using the pretrained weight you provide, and  it can work well. I get the mAP with "0.68". But when I train without pretrained weight, the mAP can only get "0.1". Can you guide me how to train from zero(without pretrained weight)?

My dataset is one class(person), it has 15000 pictures. Train : Test = 8:2
